File: 451A.py
grid=map(int,raw_input().split())
count=0
net=grid[0]*grid[1]
while(net>=1):
	net=(grid[0]*grid[1])-(grid[0]+grid[1]-1)
	grid[0]-=1
	grid[1]-=1
	count+=1
if(count%2!=0):
	print "Akshat"
else:
	print "Malvika"		

File: 451A.cpp
#include <iostream>
using namespace std;

int main()
{
	int n,m,count=0,net=1;
	cin>>n>>m;
	while(net>=1)
	{
		net=(n-1)*(m-1);
		n--;
		m--;
		count++;
	}
	if (count%2!=0)
	{
		cout<<"Akshat"<<endl;
	}
	else
	{
		cout<<"Malvika"<<endl;
	}
	return 0;
}